ocp_resources/virtual_machine.py

```diff
@@ -42,6 +42,7 @@ def __init__(
         privileged_client=None,
         yaml_file=None,
         delete_timeout=TIMEOUT_4MINUTES,
+        **kwargs,
```

Comment:
This should be added to all the classes in this PR

Reply:
Right, @Anatw will cherry-pick and add them